src/add-simple-form-value.js

```diff
@@ -21,5 +24,23 @@ export function addSimpleFormValue(value, options) {
     triplesToAdd.push(newTriple);
   }
 
+  // Add missing FormDataNode types for subjects without (rdf:type form:FormDataNode)
+  const triplesToAddLength = triplesToAdd.length;
+
+  for (let index = 0; index < triplesToAddLength; index++) {
+    const triple = triplesToAdd[index];
+
+    if (triple.subject?.value?.startsWith(URI_TEMPLATE)) {
```

Comment:
I see the following potential paths of conflicts: 

- https://github.com/lblod/submission-form-helpers/blob/master/src/generators.js
   -> uses it's own value of string so it can get out of sync
- later maybe we want to define the format of the uri in the form, not hard coded in an env variable
-  Suppose the the generators ran, and added types. And suppose `triple.subject?.value?.startsWith(URI_TEMPLATE)` matches. Do we want to add it as `formDataNode`? I have no idea...
- maybe it would be better to keep a list of the added subjects here somewhere near here. https://github.com/lblod/submission-form-helpers/blob/master/src/triples-for/triples-for-unscoped-path.js#L106 where the nodes get added. 

Anyway; things for a deeper refactor. Put these notes maybe in a comment as a `TODO`

Reply:
Also did you check this is the only exposed function that adds values?

Reply:
> Also did you check this is the only exposed function that adds values?

yes